packages/scripts/internationalization/README.md

```diff
@@ -92,6 +93,25 @@ There are 3 methods of extraction
 | files | The list of files to extract. |
 | target | The folder where the script will gather the translation files. This is used to create the zip file. Note that the files folder hierarchy is preserved.  |
 
+*Expression*
+
+```json
+{
+  "extract": {
+    "method": "expression",
+    "rootPath": "./src",
+    "expression": "message*.json",
+    "target": "./i18n"
+  }
+}
+```
+
+| Configuration | Description |
+|---|---|
+| method | `expressionn` |
+| rootPath | The folder to start the search. |
+| expression | Expression describing the files name. You can use `*` as wildcard. `message*.json` means every files that start with `message` and end with `.json` (ex: message-errors.json, message.json, ...), whereas `message.json` matches only the `message.json` files. |
```

Comment:
please provide a link to what is possible and not possible to write as expression here. regexp ? May be a link to find expression option.

Reply:
This is not a regex. If we need it, let's consider it, but for now it's only what is written here, the *, managed by find command